ethcore/vm/src/action_params.rs

```diff
@@ -130,7 +130,7 @@ impl From<ethjson::vm::Transaction> for ActionParams {
 			gas: t.gas.into(),
 			gas_price: t.gas_price.into(),
 			value: ActionValue::Transfer(t.value.into()),
-			call_type: match address.is_zero() { true => CallType::None, false => CallType::Call },	// TODO @debris is this correct?
+			call_type: if address.is_zero() { ActionType::Create } else { ActionType::Call },
```

Comment:
Probably a dumb q but is it true that a `vm::Transaction` with a `CREATE2` is indistinguishable from one with `CREATE`?

Reply:
I'm actually not quite sure whether `ActionType::Create` is right here. The type `json::vm::Transaction` is for the legacy `vmTests` in `jsontests`. It should not deal with call/create, but only the internal execution of EVM, and thus, it should always be just plain `ActionType::Call`?

Anyway, I'm not entirely sure either, but it won't matter too much -- the legacy `vmtests` is in the process of being removed and we might be able to remove this function really soon!

Reply:
My recommendation is to unconditionally use `ActionType::Call`, because that is, based on my understanding, how `vmtests` works. (I would appreciate it if someone else can take a glance on this again, just in case if I'm wrong.)

Practically as long as we're passing all `vmtests` everything should be fine.